rapyuta_io_sdk_v2/client.py

```diff
@@ -187,3 +187,1012 @@ def get_project(self, project_guid: str = None) -> Munch:
+
+        Returns:
+            Munch: List of packages as a Munch object.
+        """
+        headers = self.config.get_headers(with_project=True)
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.get(
+            url=f"{v2api_host}/v2/packages/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def create_package(self, body: dict) -> Munch:
+        """Create a new package.
+
+        The Payload is the JSON Format of the Package Manifest.
+        For a documented example, run the rio explain package command.
+
+        Returns:
+            Munch: Package details as a Munch object.
+        """
+        headers = self.config.get_headers()
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.post(
+            url=f"{v2api_host}/v2/packages/",
+            headers=headers,
+            json=body,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def get_package(self, name: str) -> Munch:
+        """Get a package by its name.
+
+        Args:
+            name (str): Package name
+
+        Returns:
+            Munch: Package details as a Munch object.
+        """
+        headers = self.config.get_headers()
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.get(
+            url=f"{v2api_host}/v2/packages/{name}/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def delete_package(self, name: str) -> Munch:
+        """Delete a package by its name.
+
+        Args:
+            name (str): Package name
+
+        Returns:
+            Munch: Package details as a Munch object.
+        """
+        headers = self.config.get_headers()
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.delete(
+            url=f"{v2api_host}/v2/packages/{name}/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def list_deployments(self, project_guid: str = None) -> Munch:
+        """List all deployments in a project.
+
+        Returns:
+            Munch: List of deployments as a Munch object.
+        """
+        headers = self.config.get_headers()
+        if project_guid:
+            headers["project"] = project_guid
+
+        v2api_host = self.config.hosts.get("v2api_host")
+
+        response = self.c.get(
+            url=f"{v2api_host}/v2/deployments/",
+            headers=headers,
+        )
+
+        handle_server_errors(response)
+
+        return munchify(response.json())
+
+    def create_deployment(self, project_guid: str, body: dict) -> Munch:
+        """Create a new deployment.
```

Comment:
suggestion: Let's use the same docstrings as Swagger docs for consistency. This docstring lacks any meaningful information that will allow a non-rapyuta.io developer to use it.

Here is the docstring from Swagger:
```md
Create a Deployment in the Project.
The Payload is the JSON Format of the Deployment Manifest.
For a documented example, run the `rio explain deployment` command.
For more examples, run the `rio list-examples` command.
```

Reply:
Ok got it. Will change for every function after changes in code.